### ISSUE TITLE
Pause indication confusing

### ISSUE THREAD

**Issue by tiktuk** [michaelvillar/timer-app#118]
A pause symbol is shown when the timer is paused which is the opposite of in (most?) music apps, where the pause symbol is shown when the music is playing. This has confused me a few time using the timer. Maybe show the text "Paused" instead?

**Comment by dwainm** [comment]
I agree that the interface could be clearer. Could we keep the pause icon but rather fade the rest of the interface to show that we're in a paused state? or bring the pause more prominent?

Along with this it would be great to have the option to disable the time, that's also adding the busyness of the interface.

**Comment by karbassi** [comment]
Closing as a design preference. The pause icon indicates the current state (paused), which is a common pattern outside of music apps. Adding text would add visual clutter to the minimal UI.
